guide/daily-spending-wallet/security.md

```diff
@@ -166,39 +158,21 @@ Bitcoin wallet users may feel hesitant about the privacy implications of your pr
+A [PIN](https://en.wikipedia.org/wiki/Personal_identification_number) (Personal identification number) is one of the most common and traditional methods for securing a mobile application. This setup requires users to choose a 4-6 digit number. They must enter it when opening their wallet application, or viewing sensitive information, such as their recovery phrase.
 
-<div class="image-slide-gallery">
-
-{% include picture.html
-   image = "/assets/images/guide/daily-spending-wallet/security/choose-pin.png"
-   retina = "/assets/images/guide/daily-spending-wallet/security/choose-pin@2x.png"
-   alt-text = "Choosing a pin screen"
-   caption = "When choosing a PIN, the user is also informed about how it is used."
-   width = 250
-   height = 541
-   layout = "shadow"
-%}
+Other application activity can also be secured via PIN entry, like approving high-value transactions, editing contacts, deleting the recovery phrase, and more.
 
-{% include picture.html
-   image = "/assets/images/guide/daily-spending-wallet/security/enter-pin.png"
-   retina = "/assets/images/guide/daily-spending-wallet/security/enter-pin@2x.png"
-   alt-text = "Enter pin screen"
-   caption = "PIN entry could be limited to a certain amount of unsuccessful attempts per day."
-   width = 250
-   height = 541
-   layout = "shadow"
-%}
-
-</div>
+{% include image-gallery.html pages = page.images_pin %}
 
-All devices can support the PIN-based approach, which makes it more accessible. A PIN can also be used to encrypt data within a wallet, making any locally stored information on the device inaccessible. Additionally, some users prefer this type of security measure because they use the same or similar PINs on multiple applications/devices, making it easy to remember.
+All devices can support the PIN-based approach, which makes it more accessible. A PIN can also be used to encrypt data within a wallet or a cloud backup, making any stored information inaccessible. If you decide to do this, ensure users understand that their wallet cannot be recovered if they lose their PIN and no other recovery mechanism is implemented.
 
 However, users with poor security practices might create easy-to-guess PINs (such as 1234), which would provide a false sense of security. This design also relies on users to remember their PIN correctly.
 
 {% include tip/open.html color="green" icon="check" label="Do: Educate on good security practices" %}
 
-During PIN creation, tell your users what a good, secure PIN is (e.g. not their birthday). Recommend against (or even make it impossible) to create a PIN such as 1234 or 1111.
+During PIN creation, tell your users what a good, secure PIN is (e.g. not their birthday). Recommend against (or even make it impossible) to create one such as 1234 or 1111.
+
+Alternative methods to PIN protection include passwords, as well as swipe patterns.
 
 {% include tip/close.html %}
 
```

Comment:
The idea of a "Brick me PIN" or wipe PIN that essentially wipes the users wallet clean (they would still have their backups hopefully) is something else worth exploring on this page.

Reply:
Do you know of any wallets that do this?

Reply:
Zeus ;) haha

Reply:
Added a note on this in the newly added "Delete wallet" paragraph.